### ISSUE TITLE
hand-rolled install.xml files.

### ISSUE THREAD

**Issue by danmarsden** [kordan/moodle-mod_surveypro#788]
The install.xml files appear to have be handwritten.

This should be fixed using the Moodle xmldb editor (under admin > development > XMLDB.)

Please hit the "load" link for your plugin install.xml file in the moodle xmldb editor, and then make Moodle think there is an update to the file (easiest way to do this is to change the description of a table - add a full stop or something similar.) Then get XMLDB to "Save" the updated file which will remove all the old invalid xml from your file. 

This is not a blocker for plugins db approval - just a nice to have.

**Comment by kordan** [comment]
Ciao Dan.
My install.xml file was 100% written using XMLDB but I was having a hard time reading it and, therefore, I stuffed it with spaces.
If it is not a blocker (and in consideration that I am the only developer of this module), I would prefer to leave it as is.

**Comment by danmarsden** [comment]
yeah - it's not a blocker, so feel free to close as won't fix, but I'd recommend you avoid adding spaces in future - it generates invalid xml and anyone that tries to install your plugin into a Totara site may hit unit test failures (as Totara includes unit tests that check the validity of the install.xml files.)
